devel/ldid-procursus/Portfile

```diff
@@ -7,9 +7,9 @@ PortGroup           makefile 1.0
 
 name                ldid-procursus
 version             2.1.5-procursus7
+revision            1
 categories          devel
 license             AGPL-3
-platforms           darwin
 conflicts           ldid
 maintainers         @therealketo procurs.us:team openmaintainer
```

Comment:
if the GitHub handle and e-mail address both belong to you then they should be withing `{}` brackets to signify that